companion/src/firmwares/opentx/opentxeeprom.h

```diff
@@ -83,7 +83,8 @@ class ProtocolsConversionTable: public ConversionTable
       addConversion(PULSES_PXX_XJT_D8, val);
       addConversion(PULSES_PXX_XJT_LR12, val++);
 
-      addConversion(PULSES_ACCESS_ISRM, val++);
+      addConversion(PULSES_ACCESS_ISRM, val);
```

Comment:
I wonder if my previous code didn't already break the module type conversion (i.e. a DSM2 module)